go/streamlog/streamlog_test.go

```diff
@@ -260,3 +264,152 @@ func TestFile(t *testing.T) {
 		t.Errorf("streamlog file: want %q got %q", want, got)
 	}
 }
+
+func TestSetAndGetMethods(t *testing.T) {
+	SetRedactDebugUIQueries(false)
```

Comment:
Whenever we change the global state in a test, we should ensure we set it back in a defer function call.

So, we should add this code to the test -

```go
originalVal := GetRedactDebugUIQueries()
defer func() {
    SetRedactDebugUIQueries(originalVal)
}()
```

We should do this for all the tests that change any state.